compiler/example/datamodel/config/policy/policy.go

```diff
@@ -1,7 +1,7 @@
 package policypkg
 
 import (
-	service_group "github.com/vmware-tanzu/graph-framework-for-microservices/compiler/example/datamodel/config/gns/service-group"
```

Comment:
would be good to have test for both named alias and case when package and directory names are different with no alias name. So maybe you can leave this import as it is and add new one without name?

Reply:
@mkczuk  I covered both scenarios


Difference in pkg and directory name **with no alias name**

[compiler/example/datamodel/config/policy/policy.go] - So removed alias name

Pls see the test added,
https://github.com/vmware-tanzu/graph-framework-for-microservices/blob/npt-608/compiler/pkg/generator/resolver_generator_test.go#L83

----


Difference in pkg and directory name **with alias name provided** 

https://github.com/vmware-tanzu/graph-framework-for-microservices/blob/npt-608/compiler/example/datamodel/config/gns/gns.go#L8 

Pls see the test added,
https://github.com/vmware-tanzu/graph-framework-for-microservices/blob/npt-608/compiler/pkg/generator/resolver_generator_test.go#L88

Reply:
ok, thanks :+1: